model/src/main/proto/spine_examples/shareaware/investment/sale_commands.proto

```diff
@@ -51,6 +51,9 @@ message SellShares {
     // The ID of the share to sell.
     ShareId share = 3 [(required) = true];
 
+    // Wanted price per one share.
```

Comment:
What we lack here is the description of the case, in which (theoretically) this "wanted" price does not correlate with the actual price on market.

I mean, the problem here is with this "wanted" word. If we say "wanted" we mean "this is the price I want my shares to be sold for", which implies the case in which "actual" price may differ (and hence, some rejections from `Market` may be thrown).

I suggest to describe this case, or find another word for "wanted".